### PR TITLE
Collect internal logs of fleet-server

### PR DESCRIPTION
**Pull request by mtojek**
Issue: https://github.com/elastic/elastic-package/issues/475

This PR extends `dump` command functionality with support for internal logs. Let's first start with fleet-server logs.

### CODE REVIEW

File: internal/stack/logs.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+		return nil, false, nil // we need to pull internal logs only from the Fleet Server container
+	}
+
+	p, err := compose.NewProject(DockerComposeProjectName)
+	if err != nil {
+		return nil, false, errors.Wrap(err, "could not create docker compose project")
+	}
+
+	serviceContainer := p.ContainerName(serviceName)
+	out, err := docker.Exec(serviceContainer, "sh", "-c", `find data/logs/default/fleet-server-json* -printf '%p\n' -exec cat {} \;`)
```
Comment (jsoriano) [inline internal/stack/logs.go:42]: Would it be an option to mount the log directories as a volume, and get the logs directly from there?

**Review by jsoriano — APPROVED** [review]
Added a question/suggestion, but it may be more complicated, current implementation LGTM.

File: internal/stack/logs.go (hunk 1 above)
Comment (mtojek) [reply internal/stack/logs.go:42]: I would have to research stability of this. In general we had issues with volume mounts if directories are recreated in runtime or mounts got lost. But I like this idea, do you think we should do it here or in a followup?

There are also other files in this directory (filebeat and metricbeat logs), but we will have to store them in internal location (like GCP bucket) as they may contain confidential data.

Comment (jsoriano) [reply internal/stack/logs.go:42]: Sure, we can do it as a follow up.

**Pull request merged**